modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring gateway API
```

Comment:
These steps install "Contour" configured for Gateway API.

Reply:
Would it make more sense to change the title to:
"Installing and configuring Contour for Gateway API"

Reply:
@bmcelvee yes, I think "Installing and configuring Contour for Gateway API" is a more appropriate title.